writeup/conStruct.tex

```diff
@@ -993,7 +994,7 @@ \subsubsection*{Poplars}
 The details of each were somewhat different 
 (Fig \ref{populus_nsp_pies} vs. Fig \ref{populus_admixture}), 
 and also differed across the replicate analyses, 
-but each strongly favored the most cluster-rich model ran 
+but each strongly favored the most cluster-rich model
```

Comment:
this phrasing is awkward.  It sounds like you mean "the highest K", but that's not what it looks like from Fig 6 (at least not "strongly")... ?